### PR TITLE
Update mailinglist link to python.org mailman list

### PR DESCRIPTION
**Pull request by jeffwidman**
Updated the mailing list urls to point to the new Flask mailing list hosted on python.org.

### CODE REVIEW

**Comment by jeffwidman** [comment]
Associated discussion thread: http://librelist.com/browser//flask/2015/7/14/switch-mailing-list-software-away-from-librelist/

Everyone who gave feedback was in favor, so we are going ahead with migrating over to the new python.org email list.

File: flask_website/templates/mailinglist/index.html [hunk 1]
```diff
@@ -4,18 +4,26 @@
...
+    fit well in a Github issue or StackOverflow question.
 
   <p>
-    To subscribe, send a mail to <em>flask@librelist.com</em> and reply
-    to the confirmation mail. Make sure to check your Spam folder, just in
-    case. To unsubscribe again, send a mail to
-    <em>flask-unsubscribe@librelist.com</em> and reply to the
-    confirmation mail.
+    You can subscribe or change your subscription options at
+    <a href="https://mail.python.org/mailman/listinfo/flask/">
```
Comment (untitaker) [inline flask_website/templates/mailinglist/index.html:13]: Can you re-add the "make sure to check spam" sentence here?

Comment (jeffwidman) [inline flask_website/templates/mailinglist/index.html:13]: Good call. Fixed and squashed into single commit.

**Comment by jeffwidman** [comment]
Anything else needed here @untitaker ? Like to get this merged so that people don't accidentally subscribe to the old list

**Comment by untitaker** [comment]
I've contacted Armin on IRC to move this repo to the `pocoo` organization so I have commit rights on it.

**Comment by jeffwidman** [comment]
Did you hear back from him yet? Or does @DasIch have commit rights to this repo?

**Comment by untitaker** [comment]
Unfortunately not, I'll ping him again on IRC.

**Pull request merged**